### ISSUE TITLE
Using the Entropy.jl in documentation instead of TransferEntropy.jl

### ISSUE THREAD

**Issue by shayandavoodii** [JuliaDynamics/TransferEntropy.jl#91]
Hi,
I was reading the great documentation of this package, and I got surprised by [this section](https://juliadynamics.github.io/TransferEntropy.jl/v1.0/#Entropies.RectangularBinning). Why you mentioned Entropies.jl instead of TransferEntropy.jl for defining the `RectangularBinning` estimator?

**Comment by kahaaga** [comment]
Hi there, @shayandavoodii! 

In Entropies.jl, we define basic functionality that is common to all information estimators in the [CausalityTools.jl](https://github.com/JuliaDynamics/CausalityTools.jl) ecosystem - transfer entropy included. TransferEntropy.jl simply re-exports some of this functionality. That's why it says `Entropies.RectangularBinning` instead of `TransferEntropy.RectangularBinning` - it's because the type is actually defined in Entropies.jl, not here. By keeping common functionality in a basic, foundations-only package, we can re-use it elsewhere, like in this package!

Binning-based probability estimation is just one of a plethora of ways to estimate probabilities (and hence transfer entropy) from data. On the main branch of Entropies.jl, I think we have more than ten different probability/entropy estimation methods now, many of which can be used to estimate transfer entropy and related measures. It's funny you ask now, because we're soon releasing Entropies v2, which will make these methods available for transfer entropy estimation too! So in future, you'll see many more exported methods from `Entropies.jl` in the documentation.

Note that we'll soon be redirecting the documentation here to [CausalityTools.jl](https://github.com/JuliaDynamics/CausalityTools.jl) instead. The reason for this is that we're moving everything in TransferEntropy.jl to CausalityTools to make code maintenance easier.

In summary: Entropies.jl defines basic functionality for probability/entropy estimation. And very soon, we're gathering all probability/entropy-based information measures such as mutual information and transfer entropy in [CausalityTools.jl](https://github.com/JuliaDynamics/CausalityTools.jl). Keep a lookout for the v2-release! (I'll try to remember to tag you here when that happens)

**Comment by shayandavoodii** [comment]
Hi, @kahaaga!
Thanks for this great explanation. I got it. So I'm going to switch on [CausalityTools](https://github.com/JuliaDynamics/CausalityTools.jl) as well!  
Great!
